sql/src/main/scala/org/apache/sedona/sql/UDF/Catalog.scala

```diff
@@ -105,6 +105,7 @@ object Catalog {
     ST_Multi,
     ST_PointOnSurface,
     ST_Reverse,
+    ST_PointN,
```

Comment:
Register ST_ExteriorRing as well

Reply:
@jiayuasu It was already implemented in Scala, so it's already present in the file